### PR TITLE
Move extension's UI to iframes

### PR DESCRIPTION
**Pull request by reepush**
I discovered following UI elements in content script:
* Password icon
screen shot 2017-08-07 at 12 45 13

* Password generation dialog
screen shot 2017-08-07 at 12 47 49

* Custom credential fields selection
screen shot 2017-08-07 at 12 46 22

### CODE REVIEW

**Comment by limpkin** [comment]
is this already fully functional and tested?

**Comment by reepush** [comment]
Of course not. Just a starting point to track progress.

**Comment by limpkin** [comment]
aha, was just making sure :)

**Comment by reepush** [comment]
With Foundation UI got this look:

screen shot 2017-08-14 at 18 25 08
screen shot 2017-08-14 at 18 25 41

**Comment by limpkin** [comment]
that's not so bad... but could we better surround the buttons so they
actually look like buttons?

**Comment by limpkin** [comment]
- confirm and cancel buttons do not work for the select custom credential fields feature
- in some cases only a blank arrow appears:
image
- the overlay on the credential fields when hovering "store or update current credentials" isn't implemented
- we may remove a little space between the "if not correct, modify it below" and the login field here:
image
- we may also add some space on the left between the popup border and each text (or simply add more margin for the complete frame)

otherwise... very nice job! I know this mustn't have been easy...

**Comment by reepush** [comment]
* confirm and cancel buttons do not work for the select custom credential fields feature
* in some cases only a blank arrow appears

Can't reproduce these issues. Could you link to the form where I can test it?

* the overlay on the credential fields when hovering "store or update current credentials" isn't implemented

I suppose you mean fields highlighting by overlay. Currently only custom fields are highlighted. Previously we highlighted fields from cipFields detection, which isn't not accurate, because we are using mcCombs for fields detection now. Need to implement simple method on mcCombs to get detected fields. Didn't find obvious way how to do it, so not implemented yet.

> we may remove a little space between the "if not correct, modify it below" and the login field here

Done.

> we may also add some space on the left between the popup border and each text (or simply add more margin for the complete frame)

Thought about that too. Done.

**Comment by reepush** [comment]
@limpkin 
Finally ready for review! [CI build](https://travis-ci.org/reepush/mooltipass/builds/265215218).

**Comment by limpkin** [comment]
thanks! seems we need to repair CI again...

File: chrome_extension/manifest.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "images/icon_password_16.png",
     "images/icon_password_24.png",
     "images/logo_inverted.png",
-    "http-auth.html"
+    "ui/*"
```
Comment (limpkin) [inline chrome_extension/manifest.json]: are you 100% you are allowed to do that?

Comment (reepush) [inline chrome_extension/manifest.json]: Yes, I tried to remove this line and iframes did't load. Also previously we had an issue with _http-auth.html_ when it wasn't copied when we pack extension and I added _ui_ folder to our build scripts.

**Comment by limpkin** [comment]
I meant... is the * wildcard actually allowed?

**Comment by reepush** [comment]
I saw one in [documentation](https://developer.chrome.com/extensions/manifest/web_accessible_resources):
```
{
  ...
  "web_accessible_resources": [
    "images/*.png",
    "style/double-rainbow.css",
```

**Comment by limpkin** [comment]
oh that's cool then!

**Pull request merged**